extensions/2.0/Vendor/NV_materials_mdl/schema/glTF.NV_materials_mdl.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "$id": "glTF.NV_materials_mdl.schema.json",
+    "title": "NV_materials_mdl glTF extension",
+    "type": "object",
+    "description": "glTF extension that enables using MDL materials.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "modules": {
+            "type": "array",
+            "items": {
+                "$ref": "module.schema.json"
+            },
+            "minItems": 1,
+            "description": "The list of all MDL modules."
+        },
+        "functionCalls": {
+            "type": "array",
+            "items": {
+                "$ref": "functionCall.schema.json"
+            },
+            "minItems": 1,
+            "description": "The list of all function calls."
+        },
+        "bsdfMeasurements": {
+            "type": "array",
+            "items": {
+                "$ref": "bsdfMeasurement.schema.json"
+            },
+            "minItems": 1,
+            "description": "The list of all BSDF measurements."
+        },
```

Comment:
`functionCall` is required in the material extension object. Should `functionCalls` array be also required here? A counter-example is using the MDL extension only to embed BSDF measurements in a glTF asset without referring to them.

Reply:
I don't think that `functionCalls` should be required for the exact reason you stated. The `functionCall` field in the material is required, but the extension object itself isn't. So, I believe not requiring any of the arrays in the glTF extension object is fine.